packages/invoices/src/OutboundShipment/DetailView/modals/utils.ts

```diff
@@ -0,0 +1,187 @@
+import { InvoiceNodeStatus } from '@openmsupply-client/common';
```

Comment:
Just all the various util fn's scattered in files - nothing new, in fact I don't think I changed anything here except for copy pasting them here.